CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -0,0 +1,31 @@
+# v1.7.0 - Changelog since v1.5.1
+
```

Comment:
Can you add a note here that the 1.6 release is not recommended due to #987 (and any other relevant issues)?

Reply:
done